src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/RemoveAzureSqlManagedDatabase.cs

```diff
@@ -0,0 +1,163 @@
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Model;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceDatabase",
+        DefaultParameterSetName = RemoveByNameAndResourceGroupParameterSet,
+        SupportsShouldProcess = true),
+        OutputType(typeof(AzureSqlManagedDatabaseModel))]
+    public class RemoveAzureSqlManagedDatabase : AzureSqlManagedDatabaseCmdletBase<IEnumerable<AzureSqlManagedDatabaseModel>>
+    {
+        protected const string RemoveByNameAndResourceGroupParameterSet =
+            "RemoveInstanceDatabaseFromInputParameters";
+
+        protected const string RemoveByInputObjectParameterSet =
+            "RemoveInstanceDatabaseFromAzureSqlManagedDatabaseModelInstanceDefinition";
+
+        protected const string RemoveByResourceIdParameterSet =
+            "RemoveInstanceDatabaseFromAzureResourceId";
+
+        /// <summary>
+        /// Gets or sets the name of the instance database to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveByNameAndResourceGroupParameterSet, 
+            Mandatory = true,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Instance Database to remove.")]
+        [Alias("InstanceDatabaseName")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the instance
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The name of the instance.")]
+        [ValidateNotNullOrEmpty]
+        public override string InstanceName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 2,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Instance database object to remove
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveByInputObjectParameterSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Instance Database object to remove")]
```

Comment:
Notice inconsistency in capitalization - "instance database" is not the name of a product, so it should not be capitalized.

Reply:
Thanks, changed everywhere